### DISCUSSION TITLE
couche raster MBtiles ne s'affiche pas sous QField

### DISCUSSION THREAD

**Discussion by geguill** [opengisch/QField#4299] [Q&A]
Bonjour à tous,
j'ai produit une couche raster au format MBtiles avec MOBAC (Mobile Atlas Creator), qui s'affiche parfaitement dans QGIS.
Mais qui ne s'affiche pas dans QField ? !
Merci de votre aide.
Cordialement
Gerard

**Comment by nirvn** [comment]
@geguill , iOS ou Android?

**Reply by geguill** [reply]
ni l'un ni l'autre, essai avec QField Windows

**Reply by nirvn** [reply]
@geguill , ok. Vous pouvez attaché un sample de votre couche ici? Ça va nous aider à faire des tests.

**Comment by geguill** [comment]
[Atlas_sans_nom_2023-05-20_085528.zip](https://github.com/opengisch/QField/files/11521754/Atlas_sans_nom_2023-05-20_085528.zip)

voici un exemple.
Merci

**Reply by nirvn** [reply]
@geguill , pouvez vous attacher un sample project (le mbtiles + qgs qui ne marche pas dans QField windows).

**Reply by geguill** [reply]
Désolé, je n'avais pas vu votre demande, avant mon post sur gpkg
[mbtiles.zip](https://github.com/opengisch/QField/files/11521826/mbtiles.zip)

**Reply by geguill** [reply]
J'ai pu tester la version Androïd : le MBtiles fonctionne (si les tuiles sont en JPG).

**Comment by geguill** [comment]
Au cas où le MBtiles ne fonctionnerait pas, j'ai vu dans l'article https://www.opengis.ch/fr/2020/06/09/offline-wms-benchmarking-raster-formats-for-qfield/, qu'il était possible de copier un fond WMS en gpkg. Dans la commande gdal_translate "WMS:url" file.xml -of WMS, où trouver l'url d'une couche WMS dans QGIS.
J'ai essayé avec l'url présente dans la fenêtre jointe, cela semble ne pas marcher.
![Capture d'écran 2023-05-20 092556](https://github.com/opengisch/QField/assets/98580457/654f54af-7944-44bf-8402-cda91c71a9b4)
Encore merci
Gérard

**Reply by geguill** [reply]
Après plusieurs essais, j'y suis arrivé !
Le format gpkg pyramid JPEG est intéressant.
La version COG d'un géotiff, préconisé dans [l'article](https://docs.qfield.org/reference/data-format/) n'est pas mal non plus. Je crois que je vais le choisir, il est plus facile à mettre en oeuvre.
Cordialement
